pkg/nvidia/build.yml

```diff
@@ -0,0 +1,7 @@
+# linuxkit build template
+#
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+#
+org: lfedge
+image: eve-nvidia
```

Comment:
Can we get this to not run as root? if you think it is possible I can tell what changes are needed.

Reply:
BTW, shouldn't we also create a repo on dockerhub where the image will be published?
https://hub.docker.com/r/lfedge/eve-nvidia does not exist...

Reply:
@shjala , this image doesn't run, it's just unpacked to the rootfs as it's included to the init section of the image yaml file (through the nvidia modifier). The only executable script (nv-init.sh) it's executed by pillar during initialization, so it runs on pillar's context...

@OhmSpectator , I've opened a ticket already at LF's Jira in order to create this repository under docker hub, so it should be ready soon.

Reply:
ah, sorry, didn't notice it

Reply:
ah :(

Reply:
so how are we going to run apps? in context of pillar process?

Reply:
That's how Pillar's initiate custom vendor stuff.... it runs anything executable under `/opt/vendor/*/init.d` during initialization: https://github.com/lf-edge/eve/blob/master/pkg/pillar/scripts/device-steps.sh#L305